api/src/team/scripts/send-organization-invitations.script.js

```diff
@@ -54,24 +56,47 @@ export class SendOrganizationInvitationsScript extends Script {
         },
       },
     });
+    this.failedLines = 0;
+    this.successLines = 0;
+    this.actualLine = 0;
```

Comment:
Le bon nom de variable à utiliser pour cela est plutôt `currentLine` : 
```suggestion
    this.currentLine = 0;
```